protocols/omega_biotek_magbind_totalpure_NGS/OmegaMagBind.ot2.py

```diff
@@ -151,10 +158,15 @@ def run_custom_protocol(
             **max_speed_per_axis)
 
     # Incubate beads and PCR product at RT for 5 minutes
+    robot.comment("Incubating the beads and PCR products at room temperature \
+for 5 minutes. Protocol will resume automatically.")
     pipette.delay(seconds=incubation_time)
 
     # Engage MagDeck and Magnetize
+    robot._driver.run_flag.wait()
     mag_deck.engage()
+    robot.comment("Delaying for "+str(settling_time)+" seconds for beads to \

```

Comment:
You have weird line endings here. Check atom to make sure you have the correct settings for line endings: https://github.com/atom/line-ending-selector